python/src/aiconfig/editor/server/server.py

```diff
@@ -114,17 +117,83 @@ def create() -> FlaskResponse:
     state.aiconfig = AIConfigRuntime.create()  # type: ignore
     return HttpResponseWithAIConfig(message="Created new AIConfig", aiconfig=state.aiconfig).to_flask_format()
 
+@app.route('/api/test_streaming', methods=["POST"])
+def test_streaming():
+    EXCLUDE_OPTIONS = {
+        "prompt_index": True,
+        "file_path": True,
+        "callback_manager": True,
+    }
```

Comment:
Isn't there a lot of duplication with `HttpResponseWithAIConfig.to_flask_format()` ?

Reply:
Yea you're right, I was mainly just playing around and trying to get things tested quickly